Sources/Dictionary/dictionaryManager.swift

```diff
@@ -0,0 +1,19 @@
+import OS
+
+public class DictionaryManager {
+	let gameWords = words
+
+	public init() {
+	}
+
+	public func getRandomIndex() -> Int {
```

Comment:
This is something I am not sure about. If I remove `public`, my tests don't work:
```
error: 'getRandomIndex' is inaccessible due to 'internal' protection level
```